### DISCUSSION TITLE
How can I change my ID?

### DISCUSSION THREAD

**Discussion by WeiyangSO** [rustdesk/rustdesk#2490] [Q&A]
Every time I change the ID, the software prompts that the server does not support it.
How to modify ID?
If it is a paid function, how much should I sponsor through github to use it?
And how to enable this function after sponsorship?? Thank you.

**Comment by HourunLi** [comment]
you can view the codes of rustdesk server. It returns "server not support" imediately after getting the tcp registerpk message.
So i guess rustdesk still doesn't support changing id
